org.lflang/src/org/lflang/validation/LFValidator.xtend

```diff
@@ -1287,11 +1288,11 @@ class LFValidator extends BaseLFValidator {
                     Literals.TYPE__STARS
                 )
             }
-        }
-        else if (this.target == Target.Python) {
+        } else if (this.target == Target.Python) {
             if (type !== null) {
                 error(
-                    "Types are not allowed in the Python target",
+                    "Types are not allowed in the Python target (found type " + type.id + 
```

Comment:
@housengw: please note this change and incorporate it in your port to Java.

Reply:
incorporated changes in commit f9741cbc6279d3d267d84f1cdd0f50c497d65749